pom.xml

```diff
@@ -86,6 +86,24 @@
             <version>1.1.1</version>
         </dependency>
     </dependencies>
+    <repositories>
```

Comment:
Good point, only it should be:
```
 <repositories>
        <repository>
            <id>DANS</id>
            <releases>
                <enabled>true</enabled>
            </releases>
            <url>http://maven.dans.knaw.nl/</url>
        </repository>
    </repositories>
    <pluginRepositories>
        <pluginRepository>
            <id>DANS</id>
            <releases>
                <enabled>true</enabled>
            </releases>
            <url>http://maven.dans.knaw.nl/</url>
        </pluginRepository>
    </pluginRepositories>
```
Those are accessible from outside the DANS network.